tests/srv6/test_srv6.py

```diff
@@ -0,0 +1,170 @@
+           " -c 'router bgp 64600 vrf Vrf10'"
+           " -c 'address-family ipv6 unicast'"
+           " -c 'sid vpn export auto'"
+           " -c 'rd vpn export 1:10'"
+           " -c 'rt vpn both 99:99'"
+           " -c 'import vpn'"
+           " -c 'export vpn'"
+           " -c 'redistribute connected'"
+           " -c 'exit'"
+           " -c 'exit'").format(V6_PREFIX_NBR, V6_MASK_NBR, V6_PREFIX_NBR, V6_MASK_NBR)
+    nbr['host'].shell(cmd)
+    Logger.info("Route %s added", V6_PREFIX_NBR)
+
+    # configure bgp on dut
+    cmd = "config vrf add Vrf10"
+    duthost.shell(cmd)
+    cmd = "config loopback add Loopback1"
+    duthost.shell(cmd)
+    cmd = "config interface vrf bind Loopback1 Vrf10"
+    duthost.shell(cmd)
+    cmd = "config interface ip add Loopback1 2001:db8:2::1"
+    duthost.shell(cmd)
+    cmd = ("vtysh"
+           " -c 'configure'"
+           " -c 'ipv6 route {}/{} Loopback1'"
+           " -c 'segment-routing'"
+           " -c 'srv6'"
+           " -c 'locators'"
+           " -c 'locator MAIN'"
+           " -c 'prefix fcbb:bbbb:2::/48 block-len 32 node-len 16 func-bits 16'"
+           " -c 'exit'"
+           " -c 'exit'"
+           " -c 'exit'"
+           " -c 'exit'"
+           " -c 'router bgp 65100'"
+           " -c 'address-family ipv6 unicast'"
+           " -c 'network {}/{}'"
+           " -c 'exit'"
+           " -c 'segment-routing srv6'"
+           " -c 'locator MAIN'"
+           " -c 'exit'"
+           " -c 'router bgp 65100 vrf Vrf10'"
+           " -c 'address-family ipv6 unicast'"
+           " -c 'sid vpn export auto'"
+           " -c 'rd vpn export 2:10'"
+           " -c 'rt vpn both 99:99'"
+           " -c 'import vpn'"
+           " -c 'export vpn'"
+           " -c 'redistribute connected'"
+           " -c 'exit'"
+           " -c 'exit'").format(V6_PREFIX_DUT, V6_MASK_DUT, V6_PREFIX_DUT, V6_MASK_DUT)
+    duthost.shell(cmd)
+    Logger.info("Route %s added to :duthost", V6_PREFIX_DUT)
+
+    # check ROUTE_TABLE
+    Logger.info("checking  DUT for route %s", V6_PREFIX_NBR)
+    cmd = "redis-cli -n 0  hgetall \"ROUTE_TABLE:Vrf10:{}\"".format(V6_PREFIX_NBR)
+    result = duthost.shell(cmd)
+    result = result['stdout']
+    Logger.info("Route table nexthops are %s", result)
+    py_assert(result != "", "The DUT did not program the SRv6 steering route")
+    py_assert("segments: Vrf10:2001:db8::/64" not in result,
+              "The DUT did not program the SRv6 steering route correctly, missing 'segments' field")
+    py_assert("source: fc00:0:1::1" not in result,
+              "The DUT did not program the SRv6 steering route correctly, missing 'source' field")
+
+    # check SRV6_SID_LIST_TABLE
+    Logger.info("checking  DUT for route %s", V6_PREFIX_NBR)
+    cmd = "redis-cli -n 0  hgetall \"SRV6_SID_LIST_TABLE:Vrf10:{}\"".format(V6_PREFIX_NBR)
+    result = duthost.shell(cmd)
+    result = result['stdout']
+    Logger.info("Route table nexthops are %s", result)
```

Comment:
The log statements seems to be not updated accordingly? You are checking SID_LIST_TABLE here right? The log statement for MY_SID table seems to be not correct as well.